cmd/agent/common/import.go

```diff
@@ -170,6 +194,12 @@ func copyFile(src, dst string, overwrite bool) error {
 	}
 	defer in.Close()
 
+	// Create necessary destination directories
+	err = os.MkdirAll(filepath.Dir(dst), 0755)
```

Comment:
Config folders should be `0750`, and files copied above should be chmoded `0640`, as they usually contain sensitive data.